resources/tests.mjs

```diff
@@ -332,6 +332,67 @@ Suites.push({
     ],
 });
 
+Suites.push({
+    name: "NewsSite-Next",
+    url: "tentative/newssite/news-next/dist/index.html",
+    async prepare(page) {
+        await page.waitForElement("#close-toast-link");
+    },
```

Comment:
Can you look whether we can have the initial rendering (including rehydration!) in the tests step instead of in the prepare phase? This would be really good for the benchmark IMO.